contrib/executor/gradle/pkg/runner/runner.go

```diff
@@ -131,9 +131,9 @@ func (r *GradleRunner) Run(ctx context.Context, execution testkube.Execution) (r
 		if args[i] == "<projectDir>" {
 			args[i] = project
 		}
+		args[i] = os.ExpandEnv(args[i])
 	}
 
-	output.PrintEvent("Running task: "+task, directory, gradleCommand, args)
```

Comment:
why do we remove this - duplication?

Reply:
Both duplication and it prints out the secret variables. The command with arguments will be printed out in the following function with obfuscated secrets, and I also checked that there are no errors before it.